packit/specfile.py

```diff
@@ -196,8 +196,21 @@ def add_patch(self, patch_metadata: PatchMetadata):
             logger.debug("There are no patches in the spec.")
             patch_number_offset = 0
 
+        if patch_metadata.patch_id is not None:
+            if patch_metadata.patch_id < patch_number_offset:
```

Comment:
I think this should be '<=', although for the '=' part `rpm` would also fail at a later point, irrc.

Reply:
this is a good point, also need to check if Patch0 is valid